recipes/fermipy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.10.4" %}
+
+package:
+  name: fermipy
+  version: {{version}}
+
+source:
+  fn: fermipy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/f/fermipy/fermipy-{{ version }}.tar.gz 
+  md5: 3eee87e3d2f324fdc51afe03f9d086a5
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+  entry_points:
+    - fermipy-dispatch = fermipy.scripts.dispatch:main
+    - fermipy-clone-configs = fermipy.scripts.clone_configs:main
+    - fermipy-collect-sources = fermipy.scripts.collect_sources:main
+    - fermipy-cluster-sources = fermipy.scripts.cluster_sources:main
+  
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - numpy >=1.6.1
+    - astropy >=1.0
+    - matplotlib >=1.4.0
+    - scipy >=0.14
+    - pyyaml
+    - healpy
+    - wcsaxes
+
+test:
+  # Python imports
+  imports:
+    - fermipy
+
+about:
+  home: http://fermipy.readthedocs.io
+  license:  BSD 3-Clause
```

Comment:
Please ask upstream to add a license file. Also please make sure it is added to `MANIFEST.in` so that it shows up in `sdist`s and other packages. Note that all BSD licenses require this thanks to the following lines.

```
1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.

2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
```